services/comm.h

```diff
@@ -61,6 +61,54 @@
 #define IS_PROTOCOL_34(c) ((c)->protocol >= 34)
 #define IS_PROTOCOL_35(c) ((c)->protocol >= 35)
 
+
+/*
+ * juliengregoire@outlook.com Patch:
+ * Create a class for reading the environement variables and replace the 
+ * the hardcoded ports !
+ *
+ */
+
+class PortCustomConfig
+{	
+	public:
+	
+	unsigned int daemonport;
+	unsigned int schedulerport;
+		
+	char* dp;
+	char* sp;
+	
+	PortCustomConfig()
+	{
+		if(getenv("daemonport") && getenv("schedulerport") != 0)
+		{
```

Comment:
I think it's a mistake made by me.

I my case I need to run multiple daemon in multiple chroot env. We have to plan to use the cgroups classes to be able to manage the cpu power of each chroot env with theere respective daemon.